datalad/tests/utils.py

```diff
@@ -147,58 +147,92 @@ def ok_clean_git_annex_proxy(path):
     """
     # TODO: May be let's make a method of AnnexRepo for this purpose
 
-    ar = AnnexRepo(path)
-    cwd = getpwd()
-    chpwd(path)
-
-    try:
-        out = ar.proxy(['git', 'status'])
-    except CommandNotAvailableError as e:
-        raise SkipTest
-    finally:
-        chpwd(cwd)
-
-    assert_in(
-        "nothing to commit", out[0],
-        msg="git-status output via proxy not plausible: %s" % out[0]
-    )
+    if isinstance(path, AnnexRepo):
+        ar = path
+    else:
+        ar = AnnexRepo(path)
```

Comment:
Right, first changed it to ease RF'ing and then melted into ok_clean_git and forgot to remove.